pkg/detectors/digitaloceanv2/digitaloceanv2.go

```diff
@@ -0,0 +1,74 @@
+package digitaloceanv2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(`\b(dop\_v1\_[a-f0-9]{64})\b`)
```

Comment:
We'll need the other two prefixes supported here as well. You can use a regex OR to do that. They'll also need to be added into the keywords

Reply:
Oh, nevermind, we’ll need to add separate verifiers for those types in later